sharded_queue/drivers/fifottl.lua

```diff
@@ -26,7 +26,7 @@ local function is_expired(task)
     return (task[index.created] + task[index.ttl]) <= time.cur()
 end
 
-local fibers_info = {}
+local wc_table = {}
```

Comment:
What does wc_table stand for?

Reply:
Wait condition table

Reply:
This comment should habe been in the code then.
I suggest to rename variable to `wait_cond_map` then.